packages/runtime-dom/src/modules/events.ts

```diff
@@ -59,16 +59,17 @@ export function patchEvent(
   }
 }
 
-const optionsModifierRE = /(?:Once|Passive|Capture)$/
+const optionsModifierRE =
+  /^(?!onOnce|onPassive|onCapture).*?(Once|Passive|Capture)$/
```

Comment:
I think this also needs to take account of events that have these names and also use modifiers. e.g. `@once.once`.

- [Playground](<https://deploy-preview-8338--vue-sfc-playground.netlify.app/#eNqNkkFPGzEQhf/KyJckEtocuKEF0aJIbQ9tBRx9WZxJMHhtyx5vIkX574ztDUkQQpzWfu9b+814duKH982QUFyJNqqgPUFESv5GWt17Fwh2EHAFe1gF18OE0Ym00ipnIwEauM72dJa1VbKKtLNg3HoaKcxgJy1ARp3B5qBKuz+jnb0zWr1ORxxNM3QmYbPU0XeknhcDWppa3EBdTZxVOJnxOd+BaaNHmm9t57VGro43hL03HSHvANqlHvIX4Daffy1FzjveJcWJ1Xz0i3AClSsPwHj/uXl2xFE5YtxTttHU/c0vNMa1c45Ysz4lIu7crcqNY3BsoRTFBqi7TM4ryno7P6lXXAiK/DArvW5eorP8/KX5UijXe20w/PP5caIUV/VZstdxis2folFIeHHQ1TOq10/0l7jNmhT/A0YMA0rx7lEX1kjVXjz8xS2v383eLZNh+gvzHnmoUs5YsZ/JLjn2CVfS/i5DrO36MS62hDYeispBM7kvvBQ82HdflH6Me9lclv94oMT+DaQaFAw=>)

Reply:
Thank you, I have fixed it in the latest commit.